projects/keycloak/secure-app-realm.json

```diff
@@ -0,0 +1,1579 @@
+[ {
```

Comment:
I think we should remove the array around the object if possible as It breaks manual importing of a realm into Keycloak, since it expects the realm config to be an object.

Reply:
sure, will do